docs/commands/Invoke-Pester.mdx

```diff
@@ -427,40 +359,53 @@ Accept wildcard characters: False
-    -- IncludeTests \<bool\>: Includes code coverage for Pester test files (*.tests.ps1).
-    Default is false.
-    One of the following: Class/Function or StartLine/EndLine
-    -- Class (C) \<string\>: Enter the class name. Wildcard characters are
-    supported, but only one string is permitted. Default is *.
-    -- Function (F) \<string\>: Enter the function name. Wildcard characters are
-    supported, but only one string is permitted. Default is *.
-    -or-
-    -- StartLine (S): Performs code coverage analysis beginning with the specified
-    line. Default is line 1.
-    -- EndLine (E): Performs code coverage analysis ending with the specified line.
-    Default is the last line of the script.
+Adds a code coverage report to the Pester tests.
+Takes strings or hash table values.
+A code coverage report lists the lines of code that did and did not run during
+a Pester test.
+This report does not tell whether code was tested; only whether
+the code ran during the test.
+By default, the code coverage report is written to the host program
+(like Write-Host).
+When you use the PassThru parameter, the custom object
+that Invoke-Pester returns has an additional CodeCoverage property that contains
+a custom object with detailed results of the code coverage test, including lines
+hit, lines missed, and helpful statistics.
+However, NUnitXml and JUnitXml output (OutputXML, OutputFormat) do not include
+any code coverage information, because it's not supported by the schema.
+Enter the path to the files of code under test (not the test file).
+Wildcard characters are supported.
+If you omit the path, the default is local
+directory, not the directory specified by the Script parameter.
+Pester test files
+are by default excluded from code coverage when a directory is provided.
+When you
+provide a test file directly using string, code coverage will be measured.
+To include
+tests in code coverage of a directory, use the dictionary syntax and provide
+IncludeTests = $true option, as shown below.
+To run a code coverage test only on selected classes, functions or lines in a script,
+enter a hash table value with the following keys:
+-- Path (P)(mandatory) \<string\>: Enter one path to the files.
+Wildcard characters
+are supported, but only one string is permitted.
+-- IncludeTests \<bool\>: Includes code coverage for Pester test files (*.tests.ps1).
+Default is false.
+One of the following: Class/Function or StartLine/EndLine
+-- Class (C) \<string\>: Enter the class name.
+Wildcard characters are
+supported, but only one string is permitted.
+Default is *.
+-- Function (F) \<string\>: Enter the function name.
+Wildcard characters are
+supported, but only one string is permitted.
+Default is *.
+-or-
+-- StartLine (S): Performs code coverage analysis beginning with the specified
+line.
+Default is line 1.
+-- EndLine (E): Performs code coverage analysis ending with the specified line.
+Default is the last line of the script.
```

Comment:
Does the hashtable-trick even work anymore? Need to update this in 5.3.0 when the CC gets some attention.

Reply:
Nope. The only thing that takes hashtables now is the New-PesterConfiguration